_elements/labels.md

```diff
@@ -22,32 +22,27 @@ lead: Intro text on what is included in this section and how to use it. No more
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
     <h4 class="usa-heading">Accessibility</h4>
-    <p>As you customize this form template, ensure it continues to follow the:</p>
-    <ul class="usa-content-list">
-      <li>Item 1</li>
-      <li>Item 2</li>
-      <li>Item 3</li>
-    </ul>
-    <p>There are no other specific accessibility guidelines for this form template.</p>
+    <p>When labels are used to call out new content that is dynamically loaded onto a page, be sure to use ARIA live regions to alert screen readers of the change. </p>
 
     <h4 class="usa-heading">Usability</h4>
     <h5>When to use</h5>
     <ul class="usa-content-list">
-      <li>Item 1</li>
-      <li>Item 2</li>
-      <li>Item 3</li>
+      <li>To draw attention to new, important content on a page that might otherwise be missed.</li>
+      <li>To filter results with one or more tags.</li>
+      <li>To indicate the number of new or unread items within a container. For example, to indicate the number of unread emails within a person’s inbox.</li>
     </ul>
-    <h5>When to consider something different</h5>
+    <h5>When to consider something else</h5>
     <ul class="usa-content-list">
-      <li>Item 1</li>
-      <li>Item 2</li>
-      <li>Item 3</li>
+      <li>When users are likely to confuse a static label with a button. For example, when the label appears in the same area of the page as buttons.</li>
+      <li>To call attention to new or updated content, consider changing the background color of the object itself or experiment with changing the font weight.</li>
+      <li>When users already expect content to be updated frequently. For example, on a site dedicated to breaking news. In this case placing the new content at the top may be enough.</li>
     </ul>
-    <h5>Guidelines</h5>
+    <h5>Guidance</h5>
     <ul class="usa-content-list">
-      <li>Item 1</li>
-      <li>Item 2</li>
-      <li>Item 3</li>
+      <li>Users frequently confuse labels as buttons. Always conduct usability testing to make sure your particular implementation is not causing frustration.</li>
+      <li>If your labels are not interactive, be sure to disable hover, focus, and active styles. </li>
+      <li>Don’t mix interactive and static labels on your site. Once you establish a pattern for how labels behave, users will expect that behavior every time. </li>
+      <li>Don’t over do it – if everything on a page is called out as important, nothing is important. </li>
```

Comment:
@colinpmacarthur can you go through and remove the extra space at the end of the list before the closing </li> tag? I think your other PR has this a few times too.